### DISCUSSION TITLE
Spigot 1.8.8 Don't work

### DISCUSSION THREAD

**Discussion by GabrielMax-1** [LuckPerms/LuckPerms#3197] [General]
Hello LuckPerms programmers, I want to report something that happened to me: when I go to install the luckperms plugin from its official page in the 1.8 Spigot / Bukkit / Paper option, when I turn on the server, the plugin does not run (let's clarify that I have spigot 1.8 .8) please solve, I like to use that plugin but it only runs in minecraft 1.16 :(, I ask you to comment on the version for that server, since, no permission plugin like permex or groupmananger has as many options as its plugin.
Thank you

**Comment by Al33013** [comment]
Me too it doesnt work L luckperm fr

**Reply by cutecat2331234** [reply]
There's a paid core plugin that supports 1.8 and offers an experience similar to LuckPerms. Would you like to try it?

**Comment by cutecat2331234** [comment]
There's a paid core plugin that supports 1.8 and offers an experience similar to LuckPerms. Would you like to try it?
